### ISSUE TITLE
Some ideas for job "motivation collars" other than for the mechanic

### ISSUE THREAD

**Issue by calzilla1** [vgstation-coders/vgstation13#28588]
A while ago #27361 was merged and added a admeme button that placed explosive collars on all living mechanics. If they did not upgrade 20 machines and/or had most APCs on the station charged or charging, ka-boom. Now, while lazy mechanics are a standout issue, I've been thinking there are other jobs that may need a bit more "motivation" to do better as well.

**Assistant:** If you job still says "assistant" on it by the end of the timer, then the collar detonates.
**Chef:** If there's not enough cooked food items to sate a certain threshold on the station for a certain amount of time, the collar self-destructs.
**Mime:** Talk=boom. Basically automating the crew's job.
**Botanist:** If you don't put at least X amount of product in the smart vendor or/and have most trays growing something after a certain time limit, then the collar separates the wheat from the chaff.
**Janitor:** If there's at least X amount of dirty tiles and/or rubbish littering the station, the collar makes a pretty big mess.
**Security Officer:** If you don't patrol at least X% of the station by the end of the timer, the collar bans your head from the server for being shitcurity.
**Warden:** If you leave brig for more than half a minute at a time, the collar goes https://rb.gy/payvxg.
**Station Engineer:** If the power grid isn't set up by a set amount of time, the collar gifts your skull to lord scrungulo.
**Cargo Technician/ Quarter Master:** If X amount of crates haven't been ordered/ accepted and shipped to the station by a set amount of time, the collar pops your head like a grape.
**Shaft Miner:** If X amount of each materials haven't been smelted, then the collar pops the weasel.
**Chemist:** If there isn't X amount of pills put into the smart vend, then the collar emits a condensed form of pacid fight into the chemists face.
**Geneticist:** If X amount of soul inhabited corpses is left in the cloning room, then they'll need a cloning for themselves. 
**Scientist:** If R&D doesn't have X amount of levels of X amount of subjects, then their collar will put their head into a destructive analyzer.

_Reminder that if any of these are encountered, the subject in question is usually a special case of laziness and dereliction of duty or have tempted ~~the gods'~~ Nanotresen's wrath_
[featurerequest]

**Comment by tomacatx2** [comment]
agreed with most of it except for mime, leave the mime slaughter to the crewmen brother

**Comment by DrSnips** [comment]
Based

**Comment by toomykins** [comment]
born to greyshit, forced to jobbie.
